ray-operator/config/samples/ray-job.tpu-v6e-256-multihost.yaml

```diff
@@ -40,7 +40,8 @@ spec:
         maxReplicas: 1
         numOfHosts: 64
         groupName: tpu-group
-        rayStartParams: {}
+        rayStartParams:
+          resources: '"{\"TPU\": 4}"'
```

Comment:
> adding resources: '"{\"TPU\": 4}"' to the rayStartParams resolves a race condition that sometimes occurs in RayServices and RayJobs where Python script execution begins before TPU device detection by the Raylets

Interesting, maybe we should consider adding TPU to rayStartParams, as in https://github.com/ray-project/kuberay/pull/2425? HDYT cc @andrewsykim

Reply:
+1, I've been testing that change with TPUs and KubeRay and can put out a PR for it in the next day or two.